eclair-core/src/main/scala/fr/acinq/eclair/db/Databases.scala

```diff
@@ -69,23 +82,122 @@ object Databases extends Logging {
         throw t
       }
     }
+  }
+
+  def postgresJDBC(database: String, host: String, port: Int,
+                   username: Option[String], password: Option[String],
+                   poolProperties: Map[String, Long],
+                   instanceId: String,
+                   databaseLeaseInterval: FiniteDuration,
+                   lockExceptionHandler: LockExceptionHandler = { _ => () },
+                   lockType: LockType = LockType.NONE, datadir: File = new File(File.separator + "tmp")): Databases = {
```

Comment:
Can we remove the default for `datadir: File`? It's not used and could lead to potential problems if the `datadir` parameter is omitted by mistake.